conf/quincy/rados/6-node-cluster-4-clients.yaml

```diff
@@ -0,0 +1,86 @@
+# Test Suite to test CIDR blocklisting of clients
+# Deployment for all the ceph daemons , with 3 mon's, 3 mgr's, 15 OSD daemons & 6 Client nodes
+# This is Openstack only test suite.
+
+globals:
+  - ceph-cluster:
+      name: ceph
+      node1:
+        networks:
+          - provider_net_cci_12
+        role:
+          - _admin
+          - mon
+          - mgr
+          - installer
+          - alertmanager
+          - grafana
+          - prometheus
+      node2:
+        networks:
+          - provider_net_cci_12
+        role:
+          - mon
+          - mgr
+          - mds
+          - rgw
+      node3:
+        networks:
+          - provider_net_cci_12
+        role:
+          - osd
+        no-of-volumes: 5
+        disk-size: 25
+      node4:
+        networks:
+          - provider_net_cci_12
+        role:
+          - osd
+        no-of-volumes: 5
+        disk-size: 25
+      node5:
+        role:
+          - osd
+        no-of-volumes: 5
+        disk-size: 25
+      node6:
+        networks:
+          - provider_net_cci_12
+        role:
+          - mon
+          - mgr
+          - mds
+          - rgw
+      node7:
+        image-name:
+          openstack: RHEL-9.3.0-x86_64-ga-latest
+          ibmc: rhel-91-server-released
```

Comment:
Why do we need to identify the Distro image?

Reply:
To let know which version of client needs to be installed.

Reply:
Why can it not be same as other nodes?

Reply:
I did not understand. what others?